PERSISTENT-WORKERS.md

```diff
@@ -37,20 +33,22 @@ Example configuration:
 ```yaml
 token: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
 
-name: "%hostname-%n"
+name: "MacMini-Slot-1"
 
 labels:
-  distro: debian
+  device: iPhone12ProMax
 ```
 
-Currently configuration files support the same set of options exposed via the command-line flags, but in the future the only way to configure certain options would be using the configuration file.
+Currently, configuration files support the same set of options exposed via the command-line flags, but in the future the only way to configure certain options would be using the configuration file.
 
 ## Writing tasks
 
 Here's an example of how to run a task on one of the persistent workers [registered in the dashboard](https://cirrus-ci.com/):
 
 ```yaml
 task:
-  persistent_worker: {}
+  persistent_worker:
+    labels:
+      os: darwin
```

Comment:
This label is reserved:

https://github.com/cirruslabs/cirrus-cli/blob/8abb5935410c7712f8204d31ff9a00879041ec96/internal/worker/worker.go#L96

Perhaps `device` can be used here too, similarly to the example above.

Reply:
What do you mean? By `device` label I meant a user label for a device that is connected to the mac-mini. Can rename to `connected-device: iPhone12ProMax`

Reply:
Oh, I see. This `label` thing matches a label instead of setting additional label for the worker to use.